<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeMasterKeyWidget</class>
 <widget class="QWidget" name="ChangeMasterKeyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>286</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="headingLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="headingLabel">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Change Master Key</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PasswordEditWidget" name="passwordEditWidget" native="true"/>
   </item>
   <item>
    <widget class="KeyFileEditWidget" name="keyFileEditWidget" native="true"/>
   </item>
   <item>
    <widget class="YubiKeyEditWidget" name="yubiKeyEditWidget" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeyFileEditWidget</class>
   <extends>QWidget</extends>
   <header>gui/masterkey/KeyFileEditWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PasswordEditWidget</class>
   <extends>QWidget</extends>
   <header>gui/masterkey/PasswordEditWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>YubiKeyEditWidget</class>
   <extends>QWidget</extends>
   <header>gui/masterkey/YubiKeyEditWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
